apps/remix-ide/src/app/providers/abstract-provider.tsx

```diff
@@ -119,7 +119,7 @@ export abstract class AbstractProvider extends Plugin implements IProvider {
       }
       this.call('notification', 'alert', modalContent)
     }
-    await this.call('udapp', 'setEnvironmentMode', {context: 'vm-merge'})
+    await this.call('udapp', 'setEnvironmentMode', {context: 'vm-paris'})
```

Comment:
why this is set as paris?

Reply:
this is a change in the ethereumjsvm api. I also changed it in our code base to avoid confusion.